code/datums/components/storage/storage.dm

```diff
@@ -15,6 +15,7 @@
 	var/datum/component/storage/concrete/master		//If not null, all actions act on master and this is just an access point.
 
 	var/list/can_hold								//if this is set, only things in this typecache will fit.
+	var/list/can_hold_extra							//if this is set, it will also be able to hold these.
```

Comment:
comment needs to be more descriptive/accurate on what this does.

Reply:
Also put the comment like this
/// Text here

as per /tg/ autodoc system standards.